models/issues/issue_list.go

```diff
@@ -599,3 +599,55 @@ func (issues IssueList) GetApprovalCounts(ctx context.Context) (map[int64][]*Rev
 
 	return approvalCountMap, nil
 }
+
+func (issues IssueList) BlockingDependenciesMap(ctx context.Context) (issueDepsMap map[int64][]*DependencyInfo, err error) {
+	var issueDeps []*DependencyInfo
+
+	err = db.GetEngine(ctx).
+		Table("issue").
+		Join("INNER", "repository", "repository.id = issue.repo_id").
+		Join("INNER", "issue_dependency", "issue_dependency.issue_id = issue.id").
+		Where(builder.In("issue_dependency.dependency_id", issues.getIssueIDs())).
+		// sort by repo id then created date
+		Asc("issue.repo_id").
+		Asc("issue.created_unix").
```

Comment:
Shouldn't this be
```suggestion
		Asc("issue.`index`").
```
?
I mean, it comes down to the same thing, but it looks more natural to me this way